lesson01.trivia/index.html

```diff
@@ -1,22 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
+<html>
 <head>
-    <title>Simple Trivia</title>
     <meta charset="utf-8">
-    <meta name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <style>
-        body {
+    <link href="style.css" type="text/css" rel="stylesheet">
+    <title>Trivia</title>
+     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-        }
-    </style>
 </head>
 <body>
-    <header>
-        <h1>Simple Trivia</h1>
-    </header>
-    <main>
-        Content Here
-    </main>
+ 
+     <div class="firstBlock" align="center" >
+               <h1 id="title">TRIVIA</h1>  
+     </div>   
+     <div class="block" align="center">
+       <!-- <div align="center"> -->
+            <p id="question" align="center" >What is the color of panda? </p>
+            <button class="button" onclick="checkAnswer(true)">Red</button><br>
```

Comment:
Not a good convention.
This means that if you decide to change the correct answer, you need to change multiple lines.
Keep the correct answer as a variable, like:
```
const correctAnswer = 3;
```